### ISSUE TITLE
[graphql-tag/loader] Allow importing named fragments in .graphql

### ISSUE THREAD

**Issue by LevanArabuli** [apollographql/graphql-tag#251]
Currently importing fragments in .graphql files imports everything, would be nice to have named imports. The problem is that Apollo Client will send all fragments with request, instead of filtering only what's actually used, eventually it results in server error `Fragment "X" is never used`.

```
// fragments.graphql
fragment info1 on Entity1 {
	id
	name
}

fragment info2 on Entity2 {
	title
	description
}
```

```
// someQuery.graphql

#import { info2 } from './fragments.graphql'
.
.
.
```

As a workaround we can split fragments into very granular files for sure, but it would be nice to have this feature.

**Comment by dobesv** [comment]
Dupe of https://github.com/apollographql/graphql-tag/issues/102 ?

**Comment by toddpadwick** [comment]
Can anyone tell me how I get this to work? I can't find specific documentation on it. 

I have queries in GQL files in which I have fragments which are also used in other gql query files.

`getEntry.gql`

I need to import a specific fragment from my `fragments.gql` file but I get a syntax error due to the import statement along with unknown fragment.

how do I use es6 js import statement inside the gql files?

**Comment by dobesv** [comment]
> how do I use es6 js import statement inside the gql files?

You can't use es6 import in gql files.  Some GraphQL setups do allow another kind of import, though.

e.g. https://www.graphql-tools.com/docs/schema-loading/

**Comment by dobesv** [comment]
I guess the fix in #102 was not actually a fix for this issue, because this issue is talking about imports written into the `.graphql` files where as #102 is talking about importing graphql files from a JavaScript file.

**Comment by clieee** [comment]
So importing specific fragments from a .gql file that contains multiple fragments will simply not work as it is right now? And will result in error message?:
```Error: Fragment "[NAME OF FRAGMENT]" is never used.```
